.github/workflows/new-linter-checklist.yaml

```
@@ -0,0 +1,35 @@
+name: Add new linter checklist
+
+on:
+  workflow_dispatch:
+    inputs:
+      pr:
+        description: PR number
+        required: true
+        type: number
+
+permissions:
+  pull-requests: write
+
+jobs:
+  add-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Add checklist
+        run: |
+          # This is just safety to not spam with new comments if the tag is
+          # removed and added back multiple times maliciously. It will reset the
+          # checklist but the previous values will be in the edit history.
+          comment_exist=$(gh pr view "$NUMBER" \
+            --json comments \
+            --jq '.comments[].author | select(.login=="github-actions") | .login' \
+            | wc -l)
+          [ "$comment_exist" -gt 0 ] && edit_last="--edit-last"
+
+          # Comment on the PR with the checklist.
+          gh pr comment "$NUMBER" --body "$(cat .github/new-linter-checklist.md)" $edit_last
```

```suggestion
          gh pr comment "$NUMBER" --body "$(cat .github/new-linter-checklist.md)"
```

We can remove all this if it's a `workflow_dispatch` and not automatically on tagging.

---

we can keep it to prevent 2 launches of the workflow dispatch in parallel.

---

but we can update the comment about the reason for this check.

---

Updated/reduced comments.